### ISSUE TITLE
Unexpected end of input when running flow-bin

### ISSUE THREAD

**Issue by dgcoffman** [creationix/http-parser-js#37]
``` bash
Error: node_modules/http-parser-js/package.json:1
  1: {
     ^ Unexpected end of input
```

When running flow.

Appears to be caused by the lines:
```
    "Jan Sch�r (https://github.com/jscissr)",
    "Paul R�tter (https://github.com/paulrutter)"
```
while
```
    "Jan Schär (https://github.com/jscissr)",
    "Paul Rütter (https://github.com/paulrutter)"
```
works fine for me.

See https://github.com/facebook/flow/issues/4867

Not sure exactly what character is supposed to be there. Maybe you can use different characters?

**Comment by grumpyoldman-io** [comment]
We're running in to this problem as well, it appears to be an encoding error of the package.json. When running in ISO encoding it doesn't fail on flow-bin.

**Comment by Jimbly** [comment]
What is throwing that error/how to reproduce this?  A new version of NPM?  Parsing the package with Node?  Some other app?

**Comment by ooflorent** [comment]
@Jimbly You can exec these commands:

```sh
mkdir flow-err
cd flow-err
yarn init -y
yarn add flow-bin http-parser-js
yarn flow init
yarn flow
```

Output:

```
Error: node_modules/http-parser-js/package.json:1
  1: {
     ^ Unexpected end of input


Found 1 error
error Command failed with exit code 2.
```

**Comment by Jimbly** [comment]
Don't have "yarn", but found if I open a terminal in UTF-8 encoding, then I see the bad characters when I cat the file (though on GitHub they look good either way), and UTF-8 seems to be the normal in Node-land, so I re-saved package.json in UTF-8 and published that as v0.4.6 to NPM a moment ago, please let me know if it fixes it.

**Comment by ooflorent** [comment]
It works now! 👍
